src/components/vue-form-builder.vue

```diff
@@ -269,7 +269,14 @@ export default {
     width: 340px;
     max-width: 340px;
     border-left: 1px solid #e9edf1;
+    overflow: hidden;
+    padding-bottom: 5%;
```

Comment:
NO NO NO.  Utilize flexbox.  Do NOT use percentages like this.

Reply:
The idea behind the bottom padding is to give a little extra spacing for the scroll.

Reply:
Percentages, ideally, should be avoided as it makes unsafe assumptions regarding viewport dimensions.  You can use calc() css statements if needed to do math more accurately.  Ideally, padding should also be based on 4s.  So 4px, 8px, 16px, 32px.